tasks/install.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Check if the desired librespot version exists
+  stat:
+    path: /opt/librespot-{{ librespot_version }}/bin/librespot
+  register: librespot_executable_stat
+
+- include_tasks: deploy.yml
+  when: not librespot_executable_stat.stat.exists
```

Comment:
If this works, you could remove the check task above.

```suggestion
  creates: "/opt/librespot-{{ librespot_version }}/bin/librespot"
```

Reply:
Does not work with the `include_tasks` module.